<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SiusDataConnections</class>
 <widget class="QDialog" name="SiusDataConnections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="portEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Ühendu</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="portSilt">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="aadressSilt">
       <property name="text">
        <string>Aadress:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="addressEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Loobu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="existingConnectionsBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Olemasolevad ühendused</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
